core/common/src/DisplayStyleSettings.ts

```diff
@@ -546,7 +548,10 @@ export class DisplayStyleSettings {
     this._monochromeMode = MonochromeMode.Flat === this._json.monochromeMode ? MonochromeMode.Flat : MonochromeMode.Scaled;
 
     this._backgroundMap = BackgroundMapSettings.fromJSON(this._json.backgroundMap);
-    this._mapImagery = MapImagerySettings.fromJSON(this._json.mapImagery, this._json.backgroundMap);
+
+    // TODO
+    // Should we have a migration path here with the old JSON
+    this._mapImagery = MapImagerySettings.fromJSON(this._json.mapImagery);
```

Comment:
How de we want to handle to migration from previous settings format?

Reply:
Perhaps a new method that does the conversion, if desired?